js/npm/Debug.hx

```diff
@@ -0,0 +1,44 @@
+package js.npm;
+
+import haxe.Log;
+import haxe.PosInfos;
+import js.Lib;
+import js.support.Either;
+
+typedef DebugFunction = Dynamic -> haxe.extern.Rest<Dynamic> -> Void;
+typedef DebugHaxeLogTraceFunction = Dynamic -> ?PosInfos -> Void;
+
+extern class Debug
+implements npm.Package.RequireNamespace<"debug","^2.2.0">
+{
+	public static inline function construct(name : String) : DebugInstance {
+		return new DebugInstance(name);
+	}
+	
+	public static inline function enable(name : String) : Void {
+		untyped Lib.require("debug").enable(name);
```

Comment:
All 4 should be possible without `inline`, since `Debug` and `DebugInstance` are two different classes, shouldn't it ?